src/registry.js

```diff
@@ -24,6 +24,12 @@ class SimpleLogger {
     arguments[0] = 'ERROR: ' + arguments[0];
     console.log.apply(console, arguments);
   }
+
+  trace() {}
```

Comment:
Why is the trace implementation different than all the other levels?

Reply:
The idea is that this logger is only used for debugging/testing. We don't want trace level messages in those cases.